pkg/innerring/processors/netmap/nodevalidation/locode/deps.go

```diff
@@ -1,49 +1,48 @@
 package locode
 
 import (
-	"github.com/nspcc-dev/neofs-node/pkg/util/locode"
-	locodedb "github.com/nspcc-dev/neofs-node/pkg/util/locode/db"
+	"github.com/nspcc-dev/locode-db/pkg/locodedb"
 )
 
 // Record is an interface of read-only
 // NeoFS LOCODE database single entry.
-type Record interface {
+type Record struct {
```

Comment:
And I'm sure we can drop it completely, record type from `locodedb` covers it completely.